_data/dtc/random.yml

```diff
@@ -0,0 +1,13 @@
+- Découverte de nos Territoires Communs
```

Comment:
I see this file is transformed to be embedded in a data attribute, but I don't see it being actually used.

Given this PR is already heavy, I'd recommend moving the work of adding the random dtc semantics to a different PR (which will also be more easily graspable by Clémentine).